lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -219,14 +219,24 @@ object NotificationHandler extends CohortHandler {
       _ <- updateCohortItemStatus(cohortItem.subscriptionName, NotificationSendComplete)
     } yield Successful
 
-  def targetAddress(contact: SalesforceContact): ZIO[Any, NotificationHandlerFailure, SalesforceAddress] =
-    (for {
-      billingAddress <- requiredField(contact.OtherAddress, "Contact.OtherAddress")
-      _ <- requiredField(billingAddress.street, "Contact.OtherAddress.street")
-      _ <- requiredField(billingAddress.city, "Contact.OtherAddress.city")
-    } yield billingAddress).orElse(
-      requiredField(contact.MailingAddress, "Contact.MailingAddress")
-    )
+  def targetAddress(
+      cohortSpec: CohortSpec,
+      contact: SalesforceContact
+  ): ZIO[Any, NotificationHandlerFailure, SalesforceAddress] = {
+    MigrationType(cohortSpec) match {
+      case DigiSubs2023 =>
+        ZIO.succeed(SalesforceAddress(Some("street"), Some("city"), Some("state"), Some("postalCode"), Some("country")))
```

Comment:
If something has to be passed through, perhaps it would be better if fields are all `None`or could have a `SalesforceAddress.Unused` whose fields are all `None`

Reply:
I tried that, as None was more natural, but that cause additional fields to fail. With that said I guess I can pass `Some("")`, that would also solve the problem and looks more.... canonical :)

Reply:
I did it here: https://github.com/guardian/price-migration-engine/pull/942/commits/499b65a73e81b28aaffc9193bbb2b0da1bd7bac2